libpod/config.go

```diff
@@ -81,13 +81,13 @@ type RootConfig struct {
 	LogDir string `toml:"log_dir"`
 
 	// FileLocking specifies whether to use file-based or in-memory locking
-	// File-based locking is required when multiple users of libkpod are
+	// File-based locking is required when multiple users of libpod are
 	// present on the same system
 	FileLocking bool `toml:"file_locking"`
 }
 
-// RuntimeConfig represents the "crio.runtime" TOML config table.
-type RuntimeConfig struct {
+// CRIOConfig represents the "crio.runtime" TOML config table.
+type CRIOConfig struct {
```

Comment:
I would be in favor of just dropping this config entirely. We only really use it in getRuntime, and it shouldn't be hard to parse the options ourselves. We might want a dedicated libpod config file similar to c/storage and c/image, but I don't think this is it.

Reply:
I think we should do this in a different PR.  We will need to supply or move the handling of crio.conf back into crio, once we remove libkpod.

Reply:
Fair enough. Can you throw a comment at the top of the file saying these configs are deprecated and going away at some point?